### ISSUE TITLE
[Request] Placable Dyed Water

### ISSUE THREAD

**Issue by Tetrajak** [KnightMiner/Inspirations#18]
It would be great if we could placed the dyed water could be picked up with a bucket and placed in the world, much like a few other mods do. If not, then integration with such mods to enable the feature would also be great!

**Comment by KnightMiner** [comment]
I cannot see myself adding any such liquid, but I might add a plugin to add recipes if colored water is detected. Will have to see if there is a common name I can use.

**Comment by Kiyaku** [comment]
Ohh cool, wasn't aware! Just checked the CT Wiki, and i see it under modtweaker, nice. 

According to the wiki at least, it mentions it consumes 1 level of dye per craft (not seeing a level value). 
I'm guessing the colored water inside cauldrons isn't an actual liquid? Otherwise i guess it could have been added as `addFluidRecipe`.

**Comment by KnightMiner** [comment]
Yeah, it's just treated as a dye. I added code to make the recipes support more than one layer, but that was after the crafttweaker support. For full compatibility you would want filling the cauldron with the dyed water to fill with dye as well, which is not currently supported though, it's something I want to look into more.
